TodoApp/TodoApp.Api/Controllers/ItemListController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Results;
+using Microsoft.Web.Http;
+using TodoApp.Api.Models;
+
+namespace TodoApp.Api.Controllers
+{
+    [ApiVersion("1")]
+    [RoutePrefix("api/v{version:apiVersion}/itemlist")]
+    public class ItemListController : ApiController
```

Comment:
Please, suffix your asynchronous methods with `Async` suffix ([reasoning](https://stackoverflow.com/a/24627122/1138663))

Reply:
Consider omitting `Item` (and its variants) suffixes. While they can be partially useful when solution grows (to distinguish between individual controllers by mere method name), it allows developers to tend to write multiple methods of a given type (e.g. `post`) to a single controller (instead of creating new controller - especially in case when there are "almost all" required references "already included" or only a subset of references is required)